pyproject.toml

```
@@ -28,4 +28,7 @@ Github = "https://github.com/ad-freiburg/qlever-control"
 [tool.setuptools]
 package-dir = { "" = "src" }
 packages = [ "qlever", "qlever.commands", "qlever.Qleverfiles" ]
```

The layout qlever uses can [automatically be discovered by setuptools](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#src-layout). The two lines can be removed.
```suggestion
[tool.setuptools]
```